app/controllers/queue_items_controller.rb

```diff
@@ -0,0 +1,43 @@
+class QueueItemsController < ApplicationController
+  before_action :require_user
+
+  def index
+    @queue_items = current_user.queue_items
+  end
+
+  def create
+    video = Video.find(params[:video_id])
+    queue_video(video)
+    redirect_to my_queue_path
+  end
+
+  def destroy
+    queue_item = QueueItem.find(params[:id])
+    queue_item.delete unless queue_item.user != current_user
+    update_queue_items_positions()
```

Comment:
Got it. Sometimes it's helpful for me to see the () to know it's a method call -- but I'm guessing that in Ruby, people leave off the () and so it'd be odd for me to code it that way since it's going against convention?

Is passing argument more flexible? i.e. is it OK to do:

```method_x(arg_y, arg_z)``` instead of ```method_x arg_y, arg_z```

I see people switch back and forth between the two.